ui.apps/src/main/content/jcr_root/apps/hclecomm/components/plugins/clientlibs/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[cq.authoring.dialog.all]"
+    dependencies="[underscore]"/>
```

Comment:
@nitheshkumar-hcl : whys do we have dependencies="[underscore] ?

Reply:
because color picker plugin js  uses js underscore dependencies like forEach.Hence provided those dependencies for color picker plugin.